pkg/helm/client.go

```diff
@@ -140,14 +142,24 @@ func (h CliClient) Install(ctx context.Context, chart, version, release, namespa
 	if wait {
 		cmd = append(cmd, "--wait")
 	}
-
+	if !dryRun {
+		out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
+		if err != nil {
+			log.Error().Print("Error installing helm chart", field.M{"output": out})
+			return "", err
+		}
+		log.Debug().Print("Result", field.M{"output": out})
+		return out, nil
+	}
+	cmd = append(cmd, "--dry-run")
+	log.Debug().Print("Executing helm install command with dry-run enabled to capture rendered manifests:")
 	out, err := RunCmdWithTimeout(ctx, h.helmBin, cmd)
 	if err != nil {
-		log.Error().Print("Error installing helm chart", field.M{"output": out})
-		return err
+		log.Error().Print("Error capturing output during Helm dry-run", field.M{"output": out, "error": err})
```

Comment:
```suggestion
		log.Error().Print("Error installing chart with dry-run enabled", field.M{"output": out, "error": err})
```
would this work, because I am assuming `RunCmdWithTimeout` actually runs the helm install command with `--dry-run` right?

Reply:
Done